modules/ocs2/src/main/scala/gem/ocs2/SmartGcalImporter.scala

```diff
@@ -80,12 +81,33 @@ object SmartGcalImporter extends TaskApp with DoobieClient {
     val y = yBinS     .parseAs(yBinning)
     val a = ampGainS  .parseAs(ampGain)
 
-    val c = SmartGcalKey.GmosNorthCommon(d, f, u, x, y, a)
+    val c = SmartGcalKey.GmosCommon(d, f, u, x, y, a)
 
     val wmin = wavelengthMinS.parseAs(Parsers.angstroms)
     val wmax = parseMaxWavelength(wavelengthMaxS)
 
-    (SmartGcalKey.GmosNorthDefinition(c, (wmin, wmax)), gcal)
+    (SmartGcalKey.GmosDefinition(c, (wmin, wmax)), gcal)
+  }
+
+  def parseGmosSouth(input: List[String]): (SmartGcalKey.GmosSouthDefinition, List[String]) = {
+    import Parsers.Gmos._
+    import Parsers.GmosSouth._
+
+    val disperserS :: filterS :: fpuS :: xBinS :: yBinS :: _ :: ampGainS :: wavelengthMinS :: wavelengthMaxS :: gcal = input
```

Comment:
Yeah it's definitely fragile, but this is only used for the initial import of string `.csv` GCAL data.  Once the data is in tables we won't need the importer anymore, I think.  The data is not edited by users but rather written out to a known format by the `exportSmartGcal` OSGi shell command I added to the ODB.  If anything goes wrong the import blows up, but that's fine I claim because this code won't be used by end-users.